test/karma-conf.mako.js

```diff
@@ -12,11 +12,13 @@ module.exports = function(config) {
   // list of files / patterns to load in the browser
   files: [
   % if mode == 'release':
+    'style/app.css',
```

Comment:
Yes because gaPrint service copy the `<link href="/lib/app.css"....>`  from the initial page.